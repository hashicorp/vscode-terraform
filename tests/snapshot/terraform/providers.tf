provider "google" {
  project = "acme-app"
  region  = "us-central1"
}
