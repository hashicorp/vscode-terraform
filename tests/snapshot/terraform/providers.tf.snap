>provider "google" {
#^^^^^^^^ source.terraform meta.block.terraform entity.name.type.terraform
#        ^ source.terraform meta.block.terraform
#         ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.begin.terraform
#          ^^^^^^ source.terraform meta.block.terraform entity.name.tag.terraform
#                ^ source.terraform meta.block.terraform entity.name.tag.terraform entity.name.tag.end.terraform
#                 ^ source.terraform meta.block.terraform
#                  ^ source.terraform meta.block.terraform punctuation.section.block.begin.terraform
>  project = "acme-app"
#^^ source.terraform meta.block.terraform
#  ^^^^^^^ source.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#         ^ source.terraform meta.block.terraform variable.declaration.terraform
#          ^ source.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#           ^ source.terraform meta.block.terraform variable.declaration.terraform
#            ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#             ^^^^^^^^ source.terraform meta.block.terraform string.quoted.double.terraform
#                     ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>  region  = "us-central1"
#^^ source.terraform meta.block.terraform
#  ^^^^^^ source.terraform meta.block.terraform variable.declaration.terraform variable.other.readwrite.terraform
#        ^^ source.terraform meta.block.terraform variable.declaration.terraform
#          ^ source.terraform meta.block.terraform variable.declaration.terraform keyword.operator.assignment.terraform
#           ^ source.terraform meta.block.terraform variable.declaration.terraform
#            ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.begin.terraform
#             ^^^^^^^^^^^ source.terraform meta.block.terraform string.quoted.double.terraform
#                        ^ source.terraform meta.block.terraform string.quoted.double.terraform punctuation.definition.string.end.terraform
>}
#^ source.terraform meta.block.terraform punctuation.section.block.end.terraform
>